examples/coupled_global_model/simple_atmos_simulation.yml

```
@@ -0,0 +1,15 @@
+z_max: 60000.0
+z_elem: 31
+dz_bottom: 50.0
+dt: "400secs"
+t_end: "1days"
+topography: "Earth"
```

```suggestion
topography: "NoWarp"
```
to remove topography